<?xml version="1.0"?>
<launch>
        <group ns="handover">
                <rosparam file="$(find bimanual_handover)/config/workspace.yaml" />
                <node name="handover_frame_publisher_node" pkg="bimanual_handover" type="handover_frame_publisher.py" output="screen" />
                <node name="workspace_analyzer_node" pkg="bimanual_handover" type="workspace_analyzer.py" output="screen" />
        </group>
        <node name="bio_ik_srv" pkg="bio_ik_service" type="bio_ik_service" output="screen" />
</launch>
